File: easy/5_check_prime/check_prime.go
package main

import "fmt"

// Check prime number
func isPrime(n int) bool {
	if n == 1 {
		return false
	}
	for i := 2; i < n; i++ {
		if n%i == 0 {
			return false
		}
	}
	return true
}

func main() {
	fmt.Println(isPrime(1))  // False
	fmt.Println(isPrime(17)) // True
}

File: easy/5_check_prime/check_prime.js
function is_prime(n) {
    if (n == 1) {
        return false;
    }
    for (let i = 2; i < n; i++) {
        if (n % i == 0) {
            return false;
        }
    }
    return true;
}

console.log(is_prime(1));  // false
console.log(is_prime(17));  // true

module.exports = is_prime;